app/styles/modules/_provider.scss

```diff
@@ -1,3 +1,4 @@
 @import 'provider/settings';
 @import 'provider/setup';
+@import 'provider/moderators';
```

Comment:
Highly recommend using ember-component-css [for route stylesheets too](https://github.com/ebryn/ember-component-css#usage-with-routes), rather than this `modules` setup.

Also, pod routes.

Reply:
Little out of scope but will create tickets because I agree with you.